Dockerfile

```
@@ -88,7 +91,7 @@ CMD ["/sbin/init"]
 
 # Install pip last to speed up local container rebuilds.
 COPY files/*.py /usr/share/container-setup/
-RUN ln -s /usr/bin/python3.11 /usr/share/container-setup/python
+RUN ln -s /usr/bin/python3.12 /usr/share/container-setup/python
```

There's an issue running the sanity tests with Python 3.12 (https://github.com/aio-libs/aiohttp/issues/7229) so I think this will need to wait.
```suggestion
RUN ln -s /usr/bin/python3.11 /usr/share/container-setup/python
```

---

@s-hertel if this is important for us, I can look into dedicating some time to supporting Python 3.12 in aiohttp.

---

@webknjaz I reverted this and marked it as resolved since we can easily work around it in the interim. Once it's fixed, we can enable the tests against 3.12 in ansible/ansible without container updates.